src/__tests__/engine/passwordless/social_or_email_login_screen.test.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+
+import { expectComponent, mockComponent } from 'testUtils';
+
+jest.mock('connection/enterprise');
+jest.mock('core/index');
+
+jest.mock('field/social/social_buttons_pane', () => mockComponent('social_buttons_pane'));
+jest.mock('field/email/email_pane', () => mockComponent('email_pane'));
```

Comment:
💪 I like the approach to just mock components instead of fighting against circular dependencies!

Reply:
Was debating whether to include your patch for circular dependencies in this PR.. what do you think?

Reply:
this is much better